zeebe/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/builder/AbstractServiceTaskBuilder.java

```diff
@@ -23,11 +23,15 @@
  * @author Sebastian Menski
  */
 public abstract class AbstractServiceTaskBuilder<B extends AbstractServiceTaskBuilder<B>>
-    extends AbstractJobWorkerTaskBuilder<B, ServiceTask> {
+    extends AbstractJobWorkerTaskBuilder<B, ServiceTask>
+    implements ZeebeExecutionListenersBuilder<B> {
```

Comment:
The current implementation of the EL Java builders is specifically tailored for ServiceTasks.
I plan to expand this support to other elements in the future, to have a more comprehensive control over the implementation as we progress.